java/src/main/java/lesson05/solution/Hello.java

```diff
@@ -0,0 +1,83 @@
+package lesson05.solution;
+
+import com.google.common.collect.ImmutableMap;
+import io.jaegertracing.internal.JaegerTracer;
+import io.opentracing.Scope;
+import io.opentracing.Tracer;
+import io.opentracing.contrib.okhttp3.TracingCallFactory;
+import io.opentracing.propagation.Format;
+import io.opentracing.tag.Tags;
+import lib.Tracing;
+import okhttp3.Call;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import java.io.IOException;
+
+public class Hello {
+
+    private final Tracer       tracer;
+    private final OkHttpClient client;
+    private final Call.Factory traceClient;
+
+    private Hello(Tracer tracer) {
+        this.tracer = tracer;
+        this.client = new OkHttpClient();
```

Comment:
doesn't look like this needs to be stored in a member var, can use local var.

Reply:
ok,modified